WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift

```diff
@@ -24,16 +29,103 @@ class JetpackScanCoordinator {
         self.view = view
     }
 
-    public func start() {
-        view.showLoading()
+    public func refreshData(showLoading: Bool = false) {
```

Comment:
Maybe instead of `showLoading` we can use `silently`?

Reply:
Reorg'd the methods a bit: d11a00b9d380b0debdfc597343cc70ae839dd684